spec/jobs/file_sets_reprocess_job_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe FileSetsReprocessJob, clean: true do
```

Comment:
As the implementation is likely something to extract to Hyku Prime, we'll want to send this file over as well.